numir/parser.js

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env node
+
+// ##EZPAARSE
+
+/*jslint maxlen: 150*/
+'use strict';
+var Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform Monographies Numérisées des bibliothèques Mathématiques Informatique Recherche des Universités Paris Diderot et Pierre et Marie Curie
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  var result = {};
+  var path   = parsedUrl.pathname;
+  // uncomment this line if you need parameters
+  var param  = parsedUrl.query || {};
+
+  // use console.error for debuging
+  // console.error(parsedUrl);
+
+  var match;
+
+  if ((match = /^\/articles\/articles.php$/.exec(path)) !== null) {
+    // http://www.numir.org/articles/articles.php?id=171&cat=5
+    result.rtype = 'BOOK';
+    result.mime  = 'PDFPLUS';
+    if (param) {
+      result.title_id = param.id || '';
+      result.unitid   = param.cat || '';
+    }
+  } else if ((match = /^\/articles\/([a-z]+).php$/.exec(path)) !== null) {
```

Comment:
Are we sure that this regex only matches bibtex ? A few lines above, we have the same route with "articles" instead of "bibtex". Maybe it should be hardcoded, since it's a condition to define the EC as REF/BIBTEX.